xbmc/GUIInfoManager.cpp

```diff
@@ -4243,393 +4248,393 @@ const infomap playlist[] =       {{ "length",           PLAYLIST_LENGTH },
 /// @{
 /// \table_start
 ///   \table_h3{ Labels, Type, Description }
-///   \table_row3{   <b>`Pvr.IsRecording`</b>,
-///                  \anchor Pvr_IsRecording
+///   \table_row3{   <b>`PVR.IsRecording`</b>,
+///                  \anchor  IsRecording
```

Comment:
Why not `PVR_IsRecoring`? We use a prefix everywhere else.

Reply:
Accidently removed, thx